### ISSUE TITLE
Select: Screen reader takes extra navigation within the Select component when use of ctrl + option right or left arrow on navigation, observed.

### ISSUE THREAD

**Issue by olehnoskov-cengage** [cengage/react-magma#2423]
Screen reader takes extra navigation within the Select component when use of ctrl + Option right or left arrow on navigation, observed. in Mac

      https://infoseci.atlassian.net/browse/AAGL-2187
      https://infoseci.atlassian.net/browse/AAGL-2189

**AAGL-2187**
**AAGL-2189**

https://github.com/user-attachments/assets/82e7095a-011f-48c6-b6bb-f5b970ad3569
https://github.com/user-attachments/assets/c631a209-66e7-418c-87a7-bfe94947b46d
https://github.com/user-attachments/assets/7a633b9d-483b-416e-a556-6c49340e43e6
https://github.com/user-attachments/assets/7b1e26d8-c8e8-4617-9e8c-f99a676e67f2

[Details](https://cengage.slack.com/archives/C8Y4FR63U/p1775565799880079)

**Comment by olehnoskov-cengage** [comment]
https://cengage.slack.com/archives/C8Y4FR63U/p1776092016994589?thread_ts=1775565799.880079&cid=C8Y4FR63U

After retesting, the Infosec team closed this ticket.
